[0;34m10:18:37  "D:\Dmall_project\case\VC_project\connect_mysql.py:46"   操作异常: (1054, "Unknown column 'supplier_code' in 'where clause'") 
 [0m[0;34m10:18:37  "D:\Dmall_project\case\DF_project\DF_supplier\conftest.py:31"   9999 
 [0m[0;34m10:18:37  "D:\Dmall_project\case\DF_project\DF_supplier\conftest.py:32"   后置操作 
 [0m[0;34m10:18:37  "D:\Dmall_project\case\DF_project\df_testlogin.py:37"   <Response [200]> 
 [0m[0;32m2020-11-03 10:18:38 - api - "D:\Dmall_project\case\DF_project\DF_supplier\supplier.py:311" - add_supplier_cards - DEBUG -  返回日志：{'code': 0, 'data': '操作成功', 'message': None, 'succ': True}[0m
